### PR TITLE
Added option to parse XPARM files instead in order to get more "indexed results"

### PR DESCRIPTION
**Pull request by asdfdsa**

### CODE REVIEW

File: edtools/extract_xds_info.py [hunk 1]
```diff
@@ -387,18 +400,18 @@ def main():
 
     if xparm:
         fns = parse_args_for_fns(args, name="XPARM.XDS", match=match)
-        foundCells = []
+        foundCells_and_Path = []
 
         for fn in fns:
             uc = parse_xparm_for_uc(fn)
-            foundCells.append(uc)
+            foundCells_and_Path.append(uc, fn)
```
Comment (stefsmeets) [inline edtools/extract_xds_info.py]: I'm pretty sure this is not valid python, can you make it `foundCells_and_Path.append((uc, fn))`?

File: edtools/update_xds.py [hunk 2]
```diff
@@ -160,10 +160,6 @@ def main():
                         action="store", type=float, dest="StrongPixel",
                         help="Parameter used for strong pixel threshold")
 
-    parser.add_argument("--match",
```
Comment (stefsmeets) [inline edtools/update_xds.py]: Why was this removed?

File: edtools/find_cell.py [hunk 3]
```diff
@@ -315,6 +315,7 @@ def put_in_order(cells):
         cell_array = np.vstack((cell_lengths, cell_angles))
         sortedArr = cell_array[:, np.argsort(cell_array[0, :])]
         sortedArr = sortedArr.ravel()
+        print(sortedArr)
```
Comment (stefsmeets) [inline edtools/find_cell.py]: Is this a debug statement? Can you remove it?

File: edtools/update_xds.py (hunk 2 above)
Comment (asdfdsa) [reply edtools/update_xds.py]: Oh, just because I guess it is a duplicate from merging, see:

https://github.com/stefsmeets/edtools/blob/63753ee3d2c01eeae61120d1e47abb818759f344/edtools/update_xds.py#L170

Comment (stefsmeets) [reply edtools/update_xds.py]: Yep, good point!

File: edtools/find_cell.py (hunk 3 above)
Comment (asdfdsa) [reply edtools/find_cell.py]: Yes! And is now removed in the latest commit :)
